src/otx/algo/modules/conv_module.py

```diff
@@ -225,7 +226,7 @@ def __init__(
         self.init_weights()
 
     @property
-    def norm(self) -> str | None:
+    def norm_layer(self) -> nn.Module | None:
```

Comment:
Why should it change? Can't we change what is implemented below?
https://github.com/openvinotoolkit/training_extensions/blob/debd5e05d9175bf47705abc6e24f8cdb61097bf9/src/otx/algo/detection/layers/csp_layer.py#L199-L203

Reply:
Unfortunately, all pretrained weights in RTDETR use "norm" for naming all norm layers. When we add "norm" as name for normalization layer it leads to recursion inside conv_module, because method norm returns layer called "norm", so they refer to each other. Because of that I renamed norm layer.

Reply:
I see. Doesn't this change affect other models?
Because this conv module is broadly used across other tasks.

Reply:
norm layer isn't called directly in models that I checked + integration tests and unit tests are passed that means there is no problem with the norm layer renaming